avocado/core/test.py

```diff
@@ -601,10 +601,11 @@ def _start_logging(self):
         self.log.addHandler(self._file_handler)
         self.log.propagate = False
 
-        # add the test log handler to the root logger so that
-        # everything logged while the test is running, for every
-        # logger, also makes its way into the test log file
-        logging.root.addHandler(self._file_handler)
+        # Adding the the test log FileHandler to the Avocado's root logger so
+        # that everything logged while the test is running, for every logger,
+        # also makes its way into the test log file
+        root = logging.getLogger('avocado')
+        root.addHandler(self._file_handler)
```

Comment:
Do you think this would affect VT tests too? If so I can perhaps do some testing here even tomorrow.

Reply:
Probably it will. Avocado-vt is using either the root logger or `__name__` (which is starting with `avocado-vt` instead `avocado.`). I was planning to use `"avocado" + __name__` as the logger name on all modules there to fix that.

If you could check if you are missing any content inside the log files with this change it would be great.

Reply:
For more context, the VT tests should be affected by this change only on the legacy runner, because the nrunner doesn't use this method.

Reply:
Thanks for narrowing this down @richtja, as we are already trying to use the nrunner and the serial runner is planned already as legacy I will do more of the testing once we have more changes in place and the ones concerning the nrunner.